include/acl_hal_mmd.h

```diff
@@ -120,6 +135,14 @@ typedef struct {
                                  int *error);
 
   double mmd_version;
+
+  // Only populated if MPSIM is active.
+  void (*aocl_mmd_simulation_streaming_kernel_info)(
```

Comment:
This should be renamed to `aocl_mmd_simulation_streaming_kernel_args()` for clarity.